<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>settingsDialog</class>
 <widget class="QWidget" name="settingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>542</width>
    <height>635</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>My Dream Diary - Settings</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="0" column="0" colspan="4">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Appearance and behavior</string>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QCheckBox" name="checkBox_4">
        <property name="whatsThis">
         <string>Saving username and login. Using this option is inadvisable when the program is used by more than one person.</string>
        </property>
        <property name="text">
         <string>Save login data</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QCheckBox" name="checkBox">
        <property name="whatsThis">
         <string>Keys can be highlighted with a selected colour. To the colour click the &quot;highlight colour...&quot;.</string>
        </property>
        <property name="text">
         <string>Highlight keys</string>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QPushButton" name="pushButton">
        <property name="whatsThis">
         <string>Keys can be highlighted with a selected colour. To the colour click the &quot;highlight colour...&quot;.</string>
        </property>
        <property name="text">
         <string>Hightlight colour...</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_3">
        <property name="whatsThis">
         <string>Select one of the icon sets occuring in the 'themes' catalog. You can also add your own sets. Details in the manual file.</string>
        </property>
        <property name="text">
         <string>Icon set:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QComboBox" name="comboBox">
        <property name="whatsThis">
         <string>Select one of the icon sets occuring in the 'themes' catalog. You can also add your own sets. Details in the manual file.</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QCheckBox" name="checkBox_7">
        <property name="whatsThis">
         <string>Tray icon allows to display information about suggestions. You can not disable this option when the suggestion system is on.</string>
        </property>
        <property name="text">
         <string>Show system tray icon</string>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QCheckBox" name="checkBox_5">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="whatsThis">
         <string>If you have decided to save your login data you can skip this process and use automatic login.</string>
        </property>
        <property name="text">
         <string>Automatic login</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QCheckBox" name="checkBox_9">
        <property name="whatsThis">
         <string>When this option is marked the program will check it's version number every time you launch it. If your program is not signified with the newest numeration you will be informed about the possibility of actualizing it. Internet connection is required.</string>
        </property>
        <property name="text">
         <string>Check updates</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0" colspan="4">
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string>Suggestions</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QCheckBox" name="checkBox_8">
        <property name="whatsThis">
         <string>It acttivates gear that finds expressions, occuring multiple times, to create a key. Details in the manual file.</string>
        </property>
        <property name="text">
         <string>Enable suggestions</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="whatsThis">
         <string>Defines the field of the researched material. Details in the manual file.</string>
        </property>
        <property name="text">
         <string>Scan data from season:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1" colspan="2">
       <widget class="QComboBox" name="comboBox_2">
        <property name="whatsThis">
         <string>Defines the field of the researched material. Details in the manual file.</string>
        </property>
        <item>
         <property name="text">
          <string>Last week</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Last month</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>All documents</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label">
        <property name="whatsThis">
         <string>Names the minimal length of the processed expressions.  Details in the manual file.</string>
        </property>
        <property name="text">
         <string>Ignore expressions shorter than:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1" colspan="2">
       <widget class="QSpinBox" name="spinBox">
        <property name="whatsThis">
         <string>Names the minimal length of the processed expressions.  Details in the manual file.</string>
        </property>
        <property name="suffix">
         <string> characters</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="label_4">
        <property name="maximumSize">
         <size>
          <width>120</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="whatsThis">
         <string>Defines the number of documetns containing the wanted expression. Details in the manual file.</string>
        </property>
        <property name="text">
         <string>Sensitivity:</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1" colspan="2">
       <widget class="QSpinBox" name="spinBox_2">
        <property name="whatsThis">
         <string>Defines the number of documetns containing the wanted expression. Details in the manual file.</string>
        </property>
        <property name="suffix">
         <string> appearance</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1" colspan="2">
       <widget class="QComboBox" name="comboBox_3">
        <property name="whatsThis">
         <string>Defines the type of the wanted expressions. Details in the manual file.</string>
        </property>
        <item>
         <property name="text">
          <string>Single and dual word</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Single word</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Dual word</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_7">
        <property name="whatsThis">
         <string>Defines the type of the wanted expressions. Details in the manual file.</string>
        </property>
        <property name="text">
         <string>Process expressions:</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0" colspan="4">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="whatsThis">
      <string>Names the types of documents which will be encoded before the saving. Details in the manual file.</string>
     </property>
     <property name="title">
      <string>Encoding</string>
     </property>
     <layout class="QGridLayout">
      <item row="0" column="0">
       <widget class="QCheckBox" name="checkBox_2">
        <property name="whatsThis">
         <string>Names the types of documents which will be encoded before the saving. Details in the manual file.</string>
        </property>
        <property name="text">
         <string>Encoding documents</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QCheckBox" name="checkBox_6">
        <property name="whatsThis">
         <string>Names the types of documents which will be encoded before the saving. Details in the manual file.</string>
        </property>
        <property name="text">
         <string>Encoding keys</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QCheckBox" name="checkBox_3">
        <property name="whatsThis">
         <string>Names the types of documents which will be encoded before the saving. Details in the manual file.</string>
        </property>
        <property name="text">
         <string>Encoding symbols</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="0" colspan="4">
    <widget class="QGroupBox" name="groupBox_4">
     <property name="title">
      <string>Browser settings</string>
     </property>
     <layout class="QGridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label_5">
        <property name="whatsThis">
         <string>The localization of the browser on your hard disk drive. If you posses several browsers the program will automatically select Mozilla Firefox.</string>
        </property>
        <property name="text">
         <string>Path:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="browserEdit">
        <property name="whatsThis">
         <string>The localization of the browser on your hard disk drive. If you posses several browsers the program will automatically select Mozilla Firefox.</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QPushButton" name="toolButton">
        <property name="whatsThis">
         <string>The localization of the browser on your hard disk drive. If you posses several browsers the program will automatically select Mozilla Firefox.</string>
        </property>
        <property name="text">
         <string>Choose...</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_6">
        <property name="whatsThis">
         <string>Internet adress of the dream-book verifying the meaning of a selected expression. The website you use must allow you to send it through the adress. The link must contain the #exp# tag, which will be replaced by your expression. The automatically selected adress is &quot;http://www.sennik.biz/?s=#exp#&quot;. More information in the manual file.</string>
        </property>
        <property name="text">
         <string>Address:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1" colspan="2">
       <widget class="QLineEdit" name="addressEdit">
        <property name="whatsThis">
         <string>Internet adress of the dream-book verifying the meaning of a selected expression. The website you use must allow you to send it through the adress. The link must contain the #exp# tag, which will be replaced by your expression. The automatically selected adress is &quot;http://www.sennik.biz/?s=#exp#&quot;. More information in the manual file.</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QPushButton" name="pushButton_3">
     <property name="text">
      <string>Help</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QPushButton" name="pushButton_2">
     <property name="whatsThis">
      <string>Restores defult settings.</string>
     </property>
     <property name="text">
      <string>Default</string>
     </property>
    </widget>
   </item>
   <item row="4" column="2">
    <widget class="QPushButton" name="okButton">
     <property name="whatsThis">
      <string>Approves settings.</string>
     </property>
     <property name="text">
      <string>Ok</string>
     </property>
    </widget>
   </item>
   <item row="4" column="3">
    <widget class="QPushButton" name="cancelButton">
     <property name="whatsThis">
      <string>Closes the 'settings' window without saving any changes.</string>
     </property>
     <property name="text">
      <string>Cancel</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
